packages/application-shell/src/components/theme-switcher/new-design-release-info-dialog.tsx

```diff
@@ -0,0 +1,36 @@
+import { FormattedMessage, useIntl } from 'react-intl';
+import { InfoDialog } from '@commercetools-frontend/application-components';
+import mcRedesignAnnouncement from '@commercetools-frontend/assets/images/mc-redesign-announcement.svg';
+import Spacings from '@commercetools-uikit/spacings';
+import Text from '@commercetools-uikit/text';
+import messages from './messages';
+
+type TNewDesignReleaseInfoDialog = {
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+const NewDesignReleaseInfoDialog = (props: TNewDesignReleaseInfoDialog) => {
+  const intl = useIntl();
+  return (
+    <InfoDialog
+      title={intl.formatMessage(messages.title)}
+      isOpen={props.isOpen}
+      onClose={props.onClose}
+      size="l"
+    >
+      <Spacings.Stack>
+        <img
+          style={{ height: '300px' }}
```

Comment:
I was wondering why do we need to set this specific height? 🤔

Reply:
Oh, there was a slight image overflow but I do not see it again. I guess it was an issue from reloading. I will update as expected.

Reply:
Thank you, updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3065/commits/6ba8ec2feb5a111e89264ca8ef8f57cf3b593cb6